underscore.js remnant code removal

```
ERROR in ./src/components/DataGrid/index.js
Module not found: Error: Cannot resolve module 'underscore' in C:\Users\soft1\headoffice\src\components\DataGrid
 @ ./src/components/DataGrid/index.js 28:18-39
```

--- a/package.json
+++ b/package.json
@@ -44,8 +44,7 @@
     "redux": "^3.5.2",
     "redux-actions": "^0.10.0",
     "redux-form": "^6.0.1",
-    "redux-thunk": "^2.1.0",
-    "underscore": "^1.8.3"
+    "redux-thunk": "^2.1.0"
   },
   "devDependencies": {
     "autoprefixer": "^6.0.3",

--- a/src/components/DataGrid/index.js
+++ b/src/components/DataGrid/index.js
@@ -1,4 +1,4 @@
-import _ from "underscore";
+import { isEmpty } from "utils";
 import React, { Component, PropTypes } from "react";
 import { DGHeaderRow } from "./DGHeaderRow";
 import { DGBody } from "./DGBody";
@@ -38,7 +38,7 @@ class DataGrid extends Component {
 
     if (isLoading) {
       return <div className={styles.spinner}><i className="fa fa-spinner fa-spin fa-2x fa-fw"/></div>;
-    } else if (_.isEmpty(rows)) {
+    } else if (isEmpty(rows)) {
       return <div className={styles.noData}>No Data Present</div>;
     }
 

